forms-flow-web/src/constants/successTypes.js

```diff
@@ -5,14 +5,22 @@ export const SL_REVIEW_SUBMISSION = "SL_REVIEW_SUBMISSION";
 export const SL_REVIEW_RESUBMISSION = "SL_REVIEW_RESUBMISSION";
 export const SL_REVIEW_FINAL_SUBMISSION = "SL_REVIEW_FINAL_SUBMISSION";
 export const COMPLAINT_INTAKE_FORM_1_10 = "1.10_COMPLAINT_INTAKE_FORM";
+export const COMPLAINT_INTAKE_FORM_1_10_LDB = "1.10_COMPLAINT_INTAKE_FORM_LDB";
 
 let submitSuccessPage = {
   seniorleadershipreview: SL_REVIEW_SUBMISSION,
   teleworkagreement: TELEWORK_SUBMISSION,
   "bullying-and-harassment-complaint-article-1-10": COMPLAINT_INTAKE_FORM_1_10,
+  "bullying-and-harassment-complaint-article-1-10-ldb": COMPLAINT_INTAKE_FORM_1_10_LDB,
```

Comment:
I realize it's not really part of this ticket and it's a bit nitpicky, but I think this could be a const rather than a let.

Reply:
Yeah! It should have been left from our legacy confirmation page :)

Reply:
Nice catch :) , updated code.